cylc/flow/cfgspec/suite.py

```diff
@@ -21,7 +21,6 @@
 
 from cylc.flow import LOG
 from cylc.flow.parsec.exceptions import UpgradeError
-from cylc.flow.network.authorisation import Priv
```

Comment:
Can remove the `Priv` class (as this PR effectively removes authorisation) and kill the `if usr_priv_level < req_priv_level:` check.